packages/dialtone-vue3/recipes/leftbar/general_row/general_row.vue

```diff
@@ -213,6 +214,14 @@ export default {
       },
     },
 
+    /**
+     * The channel setting, either 'mention' or 'always'.
+     */
```

Comment:
Add in the comment the posible values since we fill the automated docs with this:
```suggestion
     *  @values 'mention', 'always', null
     */
```
If null is not an option, then it should also be marked as `required`